lib/customer.rb

```diff
@@ -0,0 +1,68 @@
+require_relative 'menu'
+require 'rubygems'
+require 'twilio-ruby'
+require_relative '../.env.rb'
+
+class Customer
+  include Twilio
+
+  attr_accessor :menu, :new_order, :menu, :order, :total_items,
+    :order_total, :send_message, :twilio
+
+  def initialize
+    @menu = Menu.new
+    @order = {}
+    @order_total = 0
+    @total_items = 0
+  end
+
+  def receipt
+    order.map do |dish, quantity|
+      "#{quantity}x #{dish} - £#{menu.menu_list[dish] * quantity}"
+    end.join(", ")
+  end
+
+  def display_menu
+    menu.menu_list.each { |key, value| puts "#{key} - £#{value}" }
+  end
+
+  def select_dish(dish, quantity)
+      fail 'Dish not on menu' unless menu.menu_list.key?(dish)
+      update_total_items(quantity)
+      add_to_order(dish, quantity)
+      update_price(dish, quantity)
+  end
+
+  def order_total_cost
+    "Total    £" + order_total.to_s
+  end
+
+  def send_message(number_to_send_to)
+
+    twilio_sid = ENV[:account_sid]
+    twilio_token = ENV[:auth_token]
+    twilio_phone_number = ENV[:phone_num]
+
+    @twilio_client = Twilio::REST::Client.new twilio_sid, twilio_token
+
+    @twilio_client.account.messages.create(
+    :body => "Thank you! Your order was placed and will be delivered within the next hour.",
```

Comment:
Metrics/LineLength: Line is too long. [92/80]<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax.